code/__DEFINES/dcs/signals.dm

```diff
@@ -1151,8 +1151,11 @@
 // /datum/component/storage signals
 
 ///() - returns bool.
+
 #define COMSIG_CONTAINS_STORAGE "is_storage"
 ///(obj/item/inserting, mob/user, silent, force) - returns bool
+#define COMSIG_TRY_STORAGE_CONSUME_CONTENTS "storage_try_consume_contents"
+///() void signal to be sent when checking if reproductive extracts are full
```

Comment:
```suggestion
///() void signal to be sent to check if the storage is full, and if so process them
```
better clarity on what it does, it doesnt JUST check storage fullness, it also attempts to process them if it is full